_15thnight/api/account.py

```diff
@@ -66,6 +61,7 @@ def forgot_password():
     """
     Send a password reset email.
     """
+    reset_token_life = current_app.config.get("RESET_TOKEN_LIFE", "")
```

Comment:
For now, we can go with no default value as you suggest, allowing for a crash. Though in a future enhancement branch we probably should load configdist as a set of defaults. That's a really great idea.